core/src/main/scala/com/nvidia/spark/rapids/tool/Platform.scala

```diff
@@ -498,7 +497,15 @@ abstract class Platform(var gpuDevice: Option[GpuDevice],
               dynamicAllocationMinExecutors = dynamicAllocSettings.min,
               dynamicAllocationInitialExecutors = dynamicAllocSettings.initial,
               workerNodeType = Some(recommendedNodeInstance.name)
-            ))
+            )
+
+            validateRecommendedCluster(recommendedCluster) match {
+              case Right(validCluster) =>
+                recommendedNodeInstanceInfo = Some(recommendedNodeInstance)
+                recommendedClusterInfo = Some(validCluster)
+              case Left(reason) =>
+                logWarning(s"Failed to generate a cluster recommendation. Reason: $reason")
```

Comment:
Can't we add an AutoTuner comment to log that case instead of logWarning? That way, it will be kept as part of the output folder and it can be processed as part of the AutoTuner's output.
We know that the log messages are ignored when it comes to justification of output.

Reply:
That makes sense. Updated the logic to add it as AutoTuner comment.